sw/host/opentitanlib/src/transport/hyperdebug/mod.rs

```diff
@@ -469,3 +483,7 @@ impl Flavor for StandardFlavor {
         PID_HYPERDEBUG
     }
 }
+
+lazy_static! {
+    static ref SPI_REGEX: Regex = Regex::new("^ +([0-9]+) ([^ ]+) ([0-9]+)").unwrap();
+}
```

Comment:
FYI, you could have left this as a local inside of the `spi` function.

Reply:
I did not realize that, but actually, I foresee this same expression also being used from inside spi.rs in this directory, so I may want to make it `pub` here.